localenv/happy-life-bank/docker-compose.yml

```diff
@@ -58,14 +59,16 @@ services:
       AUTH_SERVER_GRANT_URL: ${HAPPY_LIFE_BANK_AUTH_SERVER_DOMAIN:-http://happy-life-bank-auth:3006}
       AUTH_SERVER_INTROSPECTION_URL: http://happy-life-bank-auth:3007
       ILP_ADDRESS: test.happy-life-bank
+      ILP_CONNECTOR_ADDRESS: http://127.0.0.1:4002
```

Comment:
While we are renaming, should we rename this to `ILP_CONNECTOR_URL` just like we have e.g. `OPEN_PAYMENTS_URL`?

Reply:
@mkurapov  👆

Reply:
`ILP_CONNECTOR_URL` makes sense to me :+1: